tests/README.md

```diff
@@ -2,31 +2,53 @@
 
 ## Installation
 
-## Enable required repositories on CentOS Stream 9
+The integration tests use the RESTful API of [podman](https://podman.io/getting-started/installation) to isolate hirte
+and the agents on multiple, containerized nodes. Therefore, a working installation of podman is required. Please refer
+to [podman installation instructions](https://podman.io/getting-started/installation).
 
-Following repositories are required on CentOS Stream 9 for successful installation:
+### Installing packages using RPM
+
+First, enable required repositories on CentOS Stream 9:
 
 ```shell
-dnf install -y dnf-plugin-config-manager
-dnf config-manager -y --set-enabled crb
-dnf install -y epel-release
+sudo dnf install -y dnf-plugin-config-manager
+sudo dnf install -y --set-enabled crb
+sudo dnf install -y epel-release
 ```
 
-## Install packages
-
-Following packages need to be installed for integration tests execution:
+Then install the required packages:
 
 ```shell
 dnf install \
     createrepo_c \
-    podman \
```

Comment:
`python3-podman` package doesn't require `podman`, so it needs to stay as we require its functionality

Reply:
Yes. I moved the installation of podman to the top of the README (below the installation section) and linked the podman installation website (https://podman.io/getting-started/installation) there. I can readd it here, though.

Reply:
OK